src/fixup.js

```diff
@@ -139,4 +140,78 @@
     }
   }
 
+  /* Deprecation warning */
+  var request = new XMLHttpRequest();
+  request.open('GET', '//www.w3.org/TR/tr-outdated-spec.json', true);
+  request.onload = function() {
+    if (request.status >= 200 && request.status < 400) {
+      var outdatedSpecs = JSON.parse(request.responseText);
+      const pathname = window.location.pathname;
+      var spec = Object.keys(outdatedSpecs).filter(function(k) {return pathname.indexOf(k) === 0;}).shift();
+      if (spec) {
+        var css = 'a#deprecationnote:hover{ background-color: transparent }'
+        ,   style = document.createElement('style');
+        if (style.styleSheet) {
+          style.styleSheet.cssText = css;
+        } else {
+          style.appendChild(document.createTextNode(css));
+        }
+        document.getElementsByTagName('head')[0].appendChild(style);
+        var nodes = document.querySelectorAll("body, h1, h2, h3");
+        for (var i = 0; i < nodes.length; i++) {
```

Comment:
As above: here web should set the class on the body to "expired" and let base.css handle it. Adding inline styles seems not great.